CPC-CHARTER.md

```diff
@@ -237,27 +237,32 @@ CPC Chairperson or Director may serve.
 ## Section 8. Board Representation
 
 There are 3 Board seats earmarked for representation from the
-OpenJS Foundation projects.  The first of these board seats is filled by
-the CPC Director as outlined in Section 7 on [Elections][] and is
-always in place.  
+OpenJS Foundation, the Foundation's projects, and related communities.
 
-The second board seat requires 2 Platinum board members to activate
-and will be filled based on processes defined
+### The Primary CPC Director (as defined in §4.3(d) in the [OpenJS Foundation bylaws][])
+
+The first of these board seats is filled as outlined in Section 7 on [Elections][] and is always in place.
```

Comment:
I assume it's okay this doesn't link anywhere?

Reply:
Nvm, assumed it would work in preview mode.

Reply:
Wait, actually, there's an extra period so it doesn't link.

Reply:
Good catch. Will update.